radiology.csl

```diff
@@ -180,6 +180,11 @@
     <choose>
       <if type="article-journal article-magazine" match="any">
         <text variable="volume"/>
+        <choose>
+          <if match="any" variable="issue">
+            <text variable="issue" prefix="(" suffix=")"/>
+          </if>
+        </choose>
```

Comment:
Thanks; wasn't clear to me from the documentation that the text node would
be skipped if the variable was empty -- I was worried about getting () in
the citation if there was no issue #.

I believe I've updated the pull request as you asked. I'm new to github, so
if it didn't come through please let me know and I'll try again.

Thanks,

John

On Fri, Jun 3, 2016 at 10:24 AM, Rintze M. Zelle <notifications@github.com>
wrote:

> In radiology.csl
> <https://github.com/citation-style-language/styles/pull/2091#discussion_r65744493>
> :
>
> > @@ -180,6 +180,11 @@
> >      <choose>
> >        <if type="article-journal article-magazine" match="any">
> >          <text variable="volume"/>
> > +        <choose>
> > +          <if match="any" variable="issue">
> > +            <text variable="issue" prefix="(" suffix=")"/>
> > +          </if>
> > +        </choose>
>
> The whole <choose/> block is unnecessary here. You can just use <text
> variable="issue" prefix="(" suffix=")"/> instead. Can you update your
> pull request?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/citation-style-language/styles/pull/2091/files/528466490745aff6f6a4bc4aafad8840d033422e#r65744493>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF3xWKnNEXXDGOp15YdE6XuXXgfrsD52ks5qIGM7gaJpZM4ItNif>
> .
>